www/src/components/Features.tsx

```diff
@@ -1,30 +1,53 @@
 import React, { FC } from 'react';
-import { FiBriefcase, FiLock, FiZap } from 'react-icons/fi';
+import { FaBatteryFull, FaSeedling } from 'react-icons/fa';
+import { FiBriefcase, FiLock, FiTerminal, FiZap } from 'react-icons/fi';
 
 const features = [
   {
     title: 'Automatic typesafety',
     description:
-      'Automatic typesafety & autocompletion inferred from your API-paths, their input data, & outputs.',
+      'Made a server side change? Typescript will warn you client side! This is inferred from your API-paths, their inputs & outputs.',
     icon: <FiLock size={20} />,
     color:
       'dark:bg-indigo-900/50 bg-indigo-200 dark:text-indigo-300 text-indigo-600',
   },
   {
-    title: 'Light & Snappy DX',
+    title: 'Snappy DX',
     description:
-      'No code generation, run-time bloat, or build pipeline. Zero dependencies & a tiny client-side footprint.',
+      'tRPC has no building or compiling steps, meaning no code generation, run-time bloat, or build pipeline. ',
```

Comment:
Is this grammatically correct?

```suggestion
      'tRPC has no build or compile steps, meaning no code generation, runtime bloat, or build pipeline.',